docs/sink_config_options.rst

```diff
@@ -10,36 +10,47 @@ Connection
   JDBC connection URL.
 
   * Type: string
-  * Default: ""
   * Importance: high
 
 ``connection.user``
   JDBC connection user.
 
   * Type: string
+  * Default: null
   * Importance: high
 
 ``connection.password``
   JDBC connection password.
 
   * Type: password
+  * Default: null
   * Importance: high
 
-DDL Support
-^^^^^^^^^^^
+Writes
+^^^^^^
 
-``auto.create``
-  Whether to automatically create the destination table based on record schema if it is found to be missing by issuing ``CREATE``.
+``insert.mode``
+  The insertion mode to use. Supported modes are:
 
-  * Type: boolean
-  * Default: false
-  * Importance: medium
+  `insert`
```

Comment:
This is not from upstream changes, but the doc for this config. Will fix, thanks!